src/App.js

```diff
@@ -67,6 +67,28 @@ const App = () => {
 			return rawNumbers.includes(rawPhone(number));
 		});
 
+	const getContactIdByNumbers = (numbers) => {
+		let id;
+		if (numbers?.length) {
+			numbers.forEach((number) => {
+				const contact = getContactByPhone(number);
+				if (contact) id = contact.contactId;
+			});
+		}
+		return id;
+	};
+
+	const getContactByNumbers = (numbers) => {
```

Comment:
Could we just use this one function to get the contact and subsequently the `id` instead of needing the function above this?